styles.css

```diff
@@ -90,29 +101,47 @@ main {
 	max-width: 2000px;
 	height: auto;
 	position: relative;
+	padding: 20px;
 }
 
 header img {
 	width: 100%;
 }
 
 .portfolio {
```

Comment:
your portfolio does not scale with the width of the page whereas your bout section does, this means about gets squished whilst the portfolio stays the same. Suggest making them adjust size together.
![image](https://user-images.githubusercontent.com/45392468/150157845-981f31c2-5665-4234-8d27-23807a08d1cf.png)

Reply:
You may want to consider adding a media query as well as the about section eventually disappears off the side of the page:
![image](https://user-images.githubusercontent.com/45392468/150158113-fadbd9f2-4abe-4703-9b96-3562e757cdcd.png)